### PR TITLE
Improved README

### PR DESCRIPTION
**Pull request by Innovatorcloudy**

### CODE REVIEW

**Review by jrobinAV — COMMENTED** [review]
Please use lines with less than 100 characters as much as possible.

**Comment by jrobinAV** [comment]
@RymMichaut I let you review this PR.

**Review by abhishek213-alb — APPROVED** [review]

**Review by RymMichaut — DISMISSED** [review]
ok for me. Longer but clearer

File: README.md [hunk 1]
```diff
@@ -76,8 +76,11 @@ To install Taipy stable release run:
 pip install taipy
 ```
 
-To install Taipy on a Conda Environment or from source, please refer to the [Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />
-To get started with Taipy, please refer to the [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/).
+Ready to Install Taipy? 🚀<br>
+Get everything set up in no time! Whether you're using a Conda environment or installing from source, follow our [Installation Guide](https://docs.taipy.io/en/latest/installation/). for step-by-step instructions.<br />
```
Comment (FlorianJacta) [inline README.md]: There is a '.' in the middle of the sentence

File: README.md [hunk 2]
```diff
@@ -76,8 +76,11 @@ To install Taipy stable release run:
...
+
+Excited to Dive In? 💡 <br>
+Start building with Taipy today! Ou [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/)  is the perfect place to begin your journey and unlock the full potential of Taipy..
```
Comment (FlorianJacta) [inline README.md]: Two points at the end of the sentence here. And you have two spaces between [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/) and 'is'

**Review by FlorianJacta — CHANGES_REQUESTED** [review]
Please, see my comments

**Review by Innovatorcloudy — COMMENTED** [review]
Sir I have updated it @FlorianJacta

**Comment by FlorianJacta** [comment]
@RymMichaut Can you recheck?

File: README.md (hunk 2 above)
Comment (FlorianJacta) [inline README.md]: Ou > Our

Comment (FlorianJacta) [reply README.md]: .. > .

File: README.md [hunk 3]
```diff
@@ -77,8 +77,14 @@ To install the Taipy stable release run:
...
 ```
 
-To install Taipy on a Conda Environment or from a source, please refer to the [Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />
-To get started with Taipy, please refer to the [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/).
+
+Ready to Install Taipy? 🚀<br>
+Get everything set up in no time! Whether you're using a Conda environment or installing from source, follow our [Installation Guide](https://docs.taipy.io/en/latest/installation/) for step-by-step instructions.<br />
+
+Excited to Dive In? 💡 <br>
+Start building with Taipy today! Our [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/) is the perfect place to begin your journey and unlock the full potential of Taipy.
```
Comment (FlorianJacta) [inline README.md]: The correct link to the Getting Started: https://docs.taipy.io/en/develop/tutorials/getting_started/

**Comment by Innovatorcloudy** [comment]
@FlorianJacta sir fixed the url

File: README.md (hunk 3 above)
Comment (jrobinAV) [reply README.md]: @FlorianJacta I believe the link should point to latest, no?

File: README.md [hunk 4]
```diff
@@ -77,8 +77,14 @@ To install the Taipy stable release run:
 pip install taipy
 ```
 
-To install Taipy on a Conda Environment or from a source, please refer to the [Installation Guide](https://docs.taipy.io/en/latest/installation/).<br />
-To get started with Taipy, please refer to the [Getting Started Guide](https://docs.taipy.io/en/latest/getting_started/).
+
+Ready to Install Taipy? 🚀<br>
+Get everything set up in no time! Whether you're using a Conda environment or installing from source, follow our [Installation Guide](https://docs.taipy.io/en/latest/installation/) for step-by-step instructions.<br />
```
Comment (jrobinAV) [inline README.md]:
```suggestion
Get everything set up in no time! Whether you're using a Conda environment or installing from
source, follow our [Installation Guide](https://docs.taipy.io/en/latest/installation/) for
step-by-step instructions.<br/>
```

File: README.md [hunk 5]
```diff
@@ -77,8 +77,14 @@ To install the Taipy stable release run:
...
+
+Excited to Dive In? 💡 <br>
+Start building with Taipy today! Our [Getting Started Guide](https://docs.taipy.io/en/develop/tutorials/getting_started/) is the perfect place to begin your journey and unlock the full potential of Taipy.
```
Comment (jrobinAV) [inline README.md]:
```suggestion
Start building with Taipy today! Our
[Getting Started Guide]https://docs.taipy.io/en/develop/tutorials/getting_started/)
is the perfect place to begin your journey and unlock the full potential of Taipy.
```

**Review by jrobinAV — CHANGES_REQUESTED** [review]

**Comment by Innovatorcloudy** [comment]
@jrobinAV sir can you check

File: README.md (hunk 3 above)
Comment (FlorianJacta) [reply README.md]: I am not sure but this link will be dead in some hours or days. I will say that it should be develop in develop and latest in latest

**Comment by Innovatorcloudy** [comment]
@FlorianJacta sir can you merge this pr

**Comment by Innovatorcloudy** [comment]
@jrobinAV Sir can you merge it too

**Review by RymMichaut — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Comment by Innovatorcloudy** [comment]
@FlorianJacta Sir can you check once

**Pull request merged**